EmbedFramework/AdaWebHost.swift

```diff
@@ -318,6 +318,16 @@ public class AdaWebHost: NSObject {
         
         debugPrint("AdaWebHost.triggerAnswer failed, webView needs to be initialized.")
     }
+    
+    public func setLanguage(language: String) {
+        if(webHostLoaded){
+            let toRun = "adaEmbed.setLanguage(\"\(language)\");"
```

Comment:
hey @lujcheng! do you think it's possible to workaround the error like this?
```suggestion
            let toRun = "adaEmbed.setLanguage(\"\(language)\"); return ''"
            
```

Reply:
It actually gives another error 
`SyntaxError: Return statements are only valid inside functions.`
It also seems a bit hacky to try to fix the issue here

Reply:
i think it makes sense to fix it here mainly because I don't think we should change Embed's API because of an issue with iOS running the Javascript.
maybe the actual Embed call needs to be wrapped in another function so that the return value that iOS expects can be provided similar to how it's done in the [Android SDK](https://github.com/AdaSupport/AndroidSDK/blob/master/ada-embed/src/main/assets/embed.html#L76)
but yeah it might be hacky for iOS since it seems to be following a different pattern for triggering the Embed functions